src/PyDynamic/uncertainty/propagate_MonteCarlo.py

```diff
@@ -339,7 +339,7 @@ def SMC(
     # Start of the actual MC part.
     print("Sequential Monte Carlo progress", end="")
 
-    for index in np.ndenumerate(x):
+    for index, _ in np.ndenumerate(x):
```

Comment:
We could also use the iterator a bit more:
```suggestion
    for index, xi in np.ndenumerate(x):
```